public-ui/src/app/data-browser/views/survey-view/components/survey-question-react.component.tsx

```diff
@@ -0,0 +1,137 @@
+import { Component, Input } from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { TooltipReactComponent } from 'app/data-browser/components/tooltip/tooltip-react.component';
+import { HighlightReactComponent } from 'app/shared/components/highlight-search/HighlightReactComponent';
+import { reactStyles } from 'app/utils';
+import { ClrIcon } from 'app/utils/clr-icon';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+import { SurveyAnswerReactComponent } from './survey-answer-react.component';
+
+
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const styles = reactStyles({
+    seeAnswers: {
+        fontSize: '14px',
+        cursor: 'pointer',
+        color: '#216fb4',
+    },
+    clIcon: {
+        width: '1.3em',
+        height: '1.3em',
+        color: '#216fb4'
+    }
+});
+
+
+interface Props {
+    isCopeSurvey: boolean;
+    question: any;
+    participantCount: number;
+    surveyConceptId: number;
+    searchTerm: string;
+}
+
+interface State {
+    showAnswers: boolean;
+    questionWithResults: any;
+}
+
+export class SurveyQuestionReactComponent extends React.Component<Props, State> {
+    constructor(props: Props, state: State) {
```

Comment:
No need for a state arg here in the constructor:
```suggestion
    constructor(props: Props) {
```